source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -46,6 +46,21 @@ def _get_shouldCreateTreeInterceptor(self):
 class Application(Document):
 	shouldCreateTreeInterceptor = False
 
+class WebDialog(Document):
+	"""

```

Comment:
Consider moving this into behaviors, as it might be useful for Edge and/or MSHTML.